From f4b3526d83c40dd8bf5948b9d7a1b2c340f0dcc8 Mon Sep 17 00:00:00 2001
From: David Howells <dhowells@redhat.com>
Date: Thu, 2 Nov 2017 15:27:48 +0000
Subject: [PATCH] afs: Connect up the CB.ProbeUuid
Git-commit: f4b3526d83c40dd8bf5948b9d7a1b2c340f0dcc8
Patch-mainline: v4.15-rc1
References: bsc#1052766

The handler for the CB.ProbeUuid operation in the cache manager is
implemented, but isn't listed in the switch-statement of operation
selection, so won't be used.  Fix this by adding it.

Signed-off-by: David Howells <dhowells@redhat.com>
Acked-by: Jan Kara <jack@suse.cz>

---
 fs/afs/cmservice.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/fs/afs/cmservice.c b/fs/afs/cmservice.c
index 24032916e0ca..03c0a8572ef4 100644
--- a/fs/afs/cmservice.c
+++ b/fs/afs/cmservice.c
@@ -121,6 +121,9 @@ bool afs_cm_incoming_call(struct afs_call *call)
 	case CBProbe:
 		call->type = &afs_SRXCBProbe;
 		return true;
+	case CBProbeUuid:
+		call->type = &afs_SRXCBProbeUuid;
+		return true;
 	case CBTellMeAboutYourself:
 		call->type = &afs_SRXCBTellMeAboutYourself;
 		return true;
-- 
2.13.6

